WebhookNotification/handler.ts

```diff
@@ -176,113 +175,116 @@ export const sendToWebhook = (
 /**
  * Returns a function for handling WebhookNotification
  */
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
 export const getWebhookNotificationHandler = (
   lNotificationModel: NotificationModel,
   notifyApiCall: TypeofApiCall<WebhookNotifyT>,
   disableWebhookMessageContent: boolean
-) => async (context: Context, input: unknown): Promise<unknown> => {
-  const inputOrError = WebhookNotificationInput.decode(input);
-
-  if (E.isLeft(inputOrError)) {
-    context.log.error(
-      `${context.executionContext.functionName}|Cannot decode input`
-    );
-    context.log.verbose(
-      `${context.executionContext.functionName}|ERROR_DETAILS=${readableReport(
-        inputOrError.left
-      )}`
-    );
-    return WebhookNotificationResult.encode({
-      kind: "FAILURE",
-      reason: "DECODE_ERROR"
-    });
-  }
+) =>
+  withJsonInput(async (context, input) => {
```

Comment:
This line is the actual change: handler function has been wrapped by `withJsonInput` decorator